client/state/action-types.js

```diff
@@ -268,6 +268,9 @@ export const SUPPORT_USER_ACTIVATE = 'SUPPORT_USER_ACTIVATE';
 export const SUPPORT_USER_ERROR = 'SUPPORT_USER_ERROR';
 export const SUPPORT_USER_TOGGLE_DIALOG = 'SUPPORT_USER_TOGGLE_DIALOG';
 export const SUPPORT_USER_TOKEN_FETCH = 'SUPPORT_USER_TOKEN_FETCH';
+export const TERMS_ADD_REQUEST = 'TERMS_ADD_REQUEST';
```

Comment:
We already have the `_REQUEST` suffix for `GET`ing items from the API, so continuing the pattern here, although repetitive, seems to make sense.